lib/galaxy/datatypes/converters/vcf_bgzip_to_vcf_converter.xml

```diff
@@ -0,0 +1,22 @@
+<tool id="CONVERTER_vcf_bgzip_to_vcf_0" name="Convert VCF_BGZIP to VCF" version="1.0.0" hidden="true">
+    <!-- <description>__NOT_USED_CURRENTLY_FOR_CONVERTERS__</description> -->
+    <requirements>
+        <requirement type="package" version="0.15.2">pysam</requirement>
+        <requirement type="package" version="3.7">python</requirement>
+    </requirements>
+    <command>python '$__tool_directory__/vcf_bgzip_to_vcf.py' '$input1' '$output1'</command>
```

Comment:
I'd just `gunzip` this, bgzip is just block-optimized and indexable but valid gzip file (http://www.htslib.org/doc/bgzip.html#BGZF_FORMAT) ... and with that said we can actually use the `CONVERTER_gz_to_uncompressed` converter.